docs/src/PolyhedralGeometry/Polyhedra/pg_polyhedra_polymake.md

```diff
@@ -0,0 +1,55 @@
+```@meta
+CurrentModule = Oscar
+```
+
+```@setup oscar
+using Oscar
+```
+
+```@contents
+Pages = ["pg_polyhedra_polymake.md"]
+```
+
+
+# `Polyhedron` and `polymake`'s `Polytope`
+
+Many polyhedral computations are done through `polymake`.
+This is visible in the structure `Polyhedron` via a pointer `pm_polytope` to the corresponding `polymake` object.
+This allows to apply all functionality of `polymake` by calling a suitable `Polymake.jl` function on this pointer.
+
+To allow both `Oscar`'s and `polymake`'s functionality to be applicable to a polyhedron object, it can be converted back and forth:
+
+```@docs
+Polyhedron(::Polymake.BigObject)
+pm_polytope(::Polyhedron)
+```
+
+The following shows all the data "known to" a `Polyhedron`.
```

Comment:
I don't see how this sentence fits with the example coming next?

Reply:
Please look at the output. https://oscar-system.github.io/Oscar.jl/previews/PR611/PolyhedralGeometry/Polyhedra/pg_polyhedra_polymake/ It displays all properties that polymake already computed of the cube. Probably the wording should be different.